posix.mak

```diff
@@ -578,10 +578,19 @@ dpl-docs: ${DUB} ${STABLE_DMD}
 	DFLAGS="$(DPL_DOCS_DFLAGS)" ${DUB} build --root=${DPL_DOCS_PATH} \
 		--compiler=${STABLE_DMD}
 
-${STABLE_DMD}:
-	mkdir -p ${STABLE_DMD_ROOT}
+# .tar.xz is archives are smaller (and don't need a temporary dir) -> prefer if available
+HAS_XZ := $(shell command -v xz 2> /dev/null)
```

Comment:
It was a bit more tricky to detect tar.xz support on older OSX versions, not sure if xz is available standalone by default nowadays.
https://github.com/dlang/installer/blob/139995801f53095af70c4393270ca86de1942e6b/script/install.sh#L110